roxctl/generate/netpol/netpol.go

```diff
@@ -0,0 +1,53 @@
+package netpol
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/printer"
+)
+
+type generateNetpolCommand struct {
+	// Properties that are bound to cobra flags.
+	offline    bool
+	folderPath string
+
+	// injected or constructed values
+	env     environment.Environment
+	printer printer.ObjectPrinter
+}
+
+// Command defines the central command tree
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	generateNetpolCmd := &generateNetpolCommand{env: cliEnvironment}
+	c := &cobra.Command{
+		Use: "netpol <folder-path>",
```

Comment:
```suggestion
		Use: "netpol <folder-path>",
		Args: cobra.ExactArgs(1),
```

Reply:
Thanks for the hint with `ExactArgs`!
For now, that is correct, but to increase robustness, there are plans to introduce more validation checks (e.g. checking if all YAMLs in the provided folder are valid) in a follow up ticket in the same Epic, so I'd lean towards keeping this for now. What do you think?